packages/sequelize-transparent-cache/lib/cache/util.js

```diff
@@ -29,20 +29,31 @@ function dataToInstance (model, data) {
   return instance
 }
 
-function loadAssociations (model) {
+function loadAssociations (model, depth = 1) {
   const associations = []
-
-  Object.keys(model.associations).forEach((key) => {
-    //  model.associations[key] does not work on include, we grab it from sequelize.model()
-    if (model.associations[key].hasOwnProperty('options')) {
-      const modelName = model.associations[key].options.name.singular
+  if (Object.keys(model.associations).length > 0 && depth <= 5) {
```

Comment:
Is there any specific meaning in number 5?

Reply:
It means it will load associations up to the fifth children. 

It is an arbitrary decision, so it does not have any specific meaning. (5 seems to cover most common cases)